BUILD/task_order/Quantum Terrarium.dat

```diff
@@ -1,20 +1,11 @@
 LX_freeCombatsTask
 catBurglarHeist
 auto_breakfastCounterVisit
-chateauPainting
-LX_faxing
-LX_artistQuest
-LX_galaktikSubQuest
-LX_armorySideQuest
-LX_meatsmithSubQuest
 L9_leafletQuest
 L5_findKnob
 L12_sonofaPrefix
 LX_burnDelay
-resolveSixthDMT
-LX_dinseylandfillFunbucks
```

Comment:
those two are also old IOTMs that currently do not work but will start working once it is out of standard. no reason to remove them